stories/theme/Map.css

```diff
@@ -0,0 +1,189 @@
+/* Mapbox styles taken from node_modules/mapbox-gl/dist/mapboxgl.css */
```

Comment:
I'd rather use the method I mentioned to @lfender6445 here. I just don't want to get anyone used to using something like this in rent-js